packages/docs/src/page-configs/styles/typography/examples/blockquote.vue

```diff
@@ -1,12 +1,10 @@
 <template>
   <blockquote class="va-blockquote">
     <p>
-      BQ: Let’s talk about meat fondue recipes and what you need to know
```

Comment:
Filler text we generally don't translate. :)

Doesn't give any info to end user.

Reply:
Thanks, I'll revert all filler text.